openvasreporting/libs/parsed_data.py

```diff
@@ -573,22 +585,23 @@ def __eq__(self, other):
             raise TypeError("Expected Vulnerability, got '{}' instead".format(type(other)))
 
         if (
-                other.vuln_id != self.vuln_id or
-                other.name != self.name or
-                other.cves != self.cves or
-                other.cvss != self.cvss or
-                other.level != self.level or
-                other.description != self.description or
-                other.detect != self.detect or
-                other.insight != self.insight or
-                other.impact != self.impact or
-                other.affected != self.affected or
-                other.solution != self.solution or
+                other.vuln_id       != self.vuln_id       or
+                other.name          != self.name          or
+                other.cves          != self.cves          or
+                other.cvss          != self.cvss          or
+                other.level         != self.level         or
+                other.description   != self.description   or
+                other.detect        != self.detect        or
+                other.version       != self.version       or
+                other.insight       != self.insight       or
+                other.impact        != self.impact        or
+                other.affected      != self.affected      or
+                other.solution      != self.solution      or
                 other.solution_type != self.solution_type or
-                other.references != self.references or
-                other.threat != self.threat or
-                other.family != self.family or
-                other.result != self.result
+                other.references    != self.references    or
+                other.threat        != self.threat        or
+                other.family        != self.family        or
+                other.result        != self.result
```

Comment:
Same comment re. PEP 8
https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements